store_queue.py

```diff
@@ -0,0 +1,55 @@
+import pika
+import settings
+
+
+class StoreQueue(object):
+
+    def __init__(self, logger):
+        self._logger = logger
+        self._connection = None
+        self._channel = None
+
+    def __connect(self):
```

Comment:
These 'private' methods should start with one underscore (http://igorsobreira.com/2010/09/16/difference-between-one-underline-and-two-underlines-in-python.html)